lm75bd/lm75bd.c

```diff
@@ -25,9 +26,46 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
+#define TEMP_REG_WRITE_BUFF_SIZE 1U
+#define TEMP_READ_BUFF_SIZE 2U
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG;
+  }
+
+  error_code_t errCode;
+
+  // Set up the buffer to store the temperature register address
+  // 0: Register address
+  uint8_t writeBuff[TEMP_REG_WRITE_BUFF_SIZE] = { LM75BD_REG_TEMP };
+
+  // Send the pointer to set up reading from the temperature register
+  errCode = i2cSendTo(devAddr, writeBuff, TEMP_REG_WRITE_BUFF_SIZE);
+  RETURN_IF_ERROR_CODE(errCode);
```

Comment:
you can just do `RETURN_IF_ERROR_CODE(i2cSendTo(devAddr, writeBuff, TEMP_REG_WRITE_BUFF_SIZE));`